core/src/utilities/helpers/generic-helpers.js

```diff
@@ -400,6 +400,28 @@ class GenericHelpersClass {
   isString(value) {
     return typeof value === 'string' || value instanceof String;
   }
+
+  removeKeyValueFromURL(urlObj, keyName) {
```

Comment:
Can you add a JSDOC type comment on this method. A bit hard to read what it does.